src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -1,20 +1,20 @@
 package frc.robot.subsystems.shooter;
 
 public interface Shooter {
-  /**
-   * Disable the speed of a motors for the shooter.
-   */
+  /** Disable the voltage of a motors for the shooter. */
   public default void disable() {}
 
-  /**
-   * Enable the speed of a motors for the shooter.
-   */
+  /** Enable the voltage of a motors for the shooter. */
   public default void enable() {}
 
   /**
-   * Set the speed of motors for the shooter.
+   * Set the voltage of motors for the shooter.
    *
-   * @param speed The speed to set. Value should be between -1.0 and 1.0.
+   * @param volts The volts to set. Value should be between -12.0 and 12.0.
    */
-  public default void setSpeed(double speed) {}
+  public default void setVoltage(double volts) {}
+
+  public double getCurrentSpeed();
+
+  public double getVoltage();
```

Comment:
these two should probably have a default implementation since the "getCurrentSpeed()" and getVoltage() is not strictly required to implement a shooter.  Also, I know Parker asked for these, but in general, if you don't have these methods implemented or have a use for them, there's no point adding it now.  You can always add these when you **actually** need them.   However, no harm in having these here.  I'm not sure how "getVoltage()" would be useful for the application to know, anyway, but hopefully Parker can chime in.